README.md

```diff
@@ -1,2 +1,12 @@
 # BeSIP
 A repository for Be-Secure Improvement proposals
+
+This repository holding multiple proposals to integrate code scanning tools in Be-Secure platform. Current set of tools include,
```

Comment:
Need not mention the tools here. What needs to go in is information about what is BeSIP ,How is it organized , what should one do to go over , contribute the BeSIP etc.